hack/create_bump_commit.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+'''
+Create a commit for boot image bump for RHCOS
+Usage: python create_bump_commit.py -v 4.6 -b 37124
+
+Requires python3-GitPython
+
+'''
+import argparse
+import git
+import os
+import subprocess
+import sys
+
+ARCH_LIST   = ['ppc64le', 's390x', 'amd64', 'all']
+URL         = 'https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos'
+VERSIONS    = ('4.6', '4.7', '4.8', '4.9', '4.10')
+COMMANDS    = []
+
+OLD_ARCH_LIST   =  ['ppc64le', 's390x', 'amd64']
+NEW_ARCH_LIST   =  OLD_ARCH_LIST.copy()
+NEW_ARCH_LIST.append('aarch64')
+
+ARCH_ALLOWED = {}
+ARCH_ALLOWED['4.6']    = OLD_ARCH_LIST
+ARCH_ALLOWED['4.7']    = OLD_ARCH_LIST
+ARCH_ALLOWED['4.8']    = NEW_ARCH_LIST
+ARCH_ALLOWED['4.9']    = NEW_ARCH_LIST
+ARCH_ALLOWED['4.10']   = NEW_ARCH_LIST
+
+def usage():
+    '''
+    # Handle parameters
+    @param version string rhcos version
+    '''
+
+    parser = argparse.ArgumentParser(
+        description='python boot-image-bumps.py -v 4.6',
+    )
+
+    parser.add_argument("-a", "--arch", choices=ARCH_LIST, default="all")
+    parser.add_argument("-b", "--bugnumber", required=True)
```

Comment:
Is this requiring a bug for every bump? What about when we're bumping during a new release and we don't need a bz?

Reply:
Thanks for checking it @patrickdillon! Done.